contracts/core/extensions/CoreInternal.sol

```diff
@@ -36,12 +36,19 @@ contract CoreInternal is
 
     /* ============ Events ============ */
 
-    // Logs registration of new exchange
+    // Logs registration of new exchange conforming to IExchangeWrapper
     event ExchangeRegistered(
         uint8 _exchangeId,
         address _exchange
     );
 
+    // Logs registration of a new factory conforming to ISetFactory
```

Comment:
Nit: More accurate to say something like "Logs a change in the registration of a factory..."